Games/gameforgeclient.yml

```diff
@@ -0,0 +1,28 @@
+Name: gameforgeclient
+Description: The official Gameforge launcher.
+Grade: Platinum
+Arch: win64
+
+Dependencies:
+- d3dx9
+- msls31
+- allfonts
+- d3dcompiler_43
+- d3dcompiler_47
+
+Parameters:
+  dxvk: true
+  sync: esync
+  discrete_gpu: true
+
+Executable:
+  name: GameforgeInstaller
+  icon: gameforge.ico
```

Comment:
please use a svg or png icon instead

Reply:
also add screenshot.png in the data folder please

Reply:
Done!